test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -2699,7 +2699,7 @@ TEST_P(SslSocketTest, ShutdownWithoutCloseNotify) {
         Buffer::OwnedImpl data("hello");
         server_connection->write(data, false);
         EXPECT_EQ(data.length(), 0);
-        // Calling close(FlushWrite) in onEvent() callback results in PostIoAction::Close,
+        // Calling close(FlushWrite) in onEvent() callback results in PostIoAction::CloseGraceful,
```

Comment:
Seeing some failures in //test/extensions/transport_sockets/tls:ssl_socket_test, examples:

2020-11-14T01:05:04.6013231Z Uninteresting mock function call - taking default action specified at:
2020-11-14T01:05:04.6013980Z test/mocks/stream_info/mocks.cc:30:
2020-11-14T01:05:04.6014503Z     Function call: setConnectionTerminationDetails("downstream connection was terminated")

2020-11-14T01:05:04.6024240Z unknown file: Failure
2020-11-14T01:05:04.6025476Z Uninteresting mock function call - taking default action specified at:
2020-11-14T01:05:04.6026043Z test/mocks/stream_info/mocks.cc:24:
2020-11-14T01:05:04.6026487Z     Function call: setResponseFlag(16384)

Reply:
Yeah, [see my previous comment](https://github.com/envoyproxy/envoy/pull/13941#issuecomment-724397165) - I didn't add mocks for them, since it's a lot of extra noise in PR for code that was still being discussed. Also, I'm not sure if all 3 values (connection termination details, upstream transport socket failure and response flags) should be set.